apps/app/src/views/routes/Order.tsx

```diff
@@ -520,43 +521,45 @@ export const Order = () => {
               Fills
             </Text>
             {order?.fills.length > 0 ? (
-              <TableContainer w="full">
-                <Table bg="transparent" size="sm">
-                  <Tbody>
-                    {order.fills.map((fill: any, i: number) => {
-                      return i < 5 ? (
-                        <Tr
-                          key={fill.id}
-                          onClick={() => navigate(`/prescriptions/${fill?.prescription?.id}`)}
-                          _hover={{ backgroundColor: 'gray.100' }}
-                          cursor="pointer"
-                        >
-                          <Td px={0} py={3} whiteSpace="pre-wrap" borderColor="gray.200">
-                            <HStack w="full" justify="space-between">
-                              <VStack alignItems="start">
-                                <HStack>
-                                  <Text>{fill.treatment.name}</Text>
-                                </HStack>
-                                <HStack>
-                                  <Badge
-                                    size="sm"
-                                    colorScheme={FILL_COLOR_MAP[fill.state as keyof object] || ''}
-                                  >
-                                    {FILL_STATE_MAP[fill.state as keyof object] || ''}
-                                  </Badge>
-                                </HStack>
-                              </VStack>
-                              <Box alignItems="end">
-                                <FiChevronRight size="1.3em" />
-                              </Box>
+              <>
+                {order.fills.map((fill: any, i: number) => {
+                  return i < 5 ? (
+                    <LinkBox key={fill.id} w="full" style={{ textDecoration: 'none' }}>
+                      <Card
+                        variant="outline"
+                        p={[2, 3]}
+                        w="full"
+                        shadow="none"
+                        _hover={{
+                          backgroundColor: 'gray.50'
+                        }}
+                      >
+                        <HStack justifyContent="space-between">
+                          <VStack alignItems="start">
+                            <HStack>
+                              <LinkOverlay href={`/prescriptions/${fill?.prescription?.id}`}>
+                                <Text>{fill.treatment.name}</Text>
+                              </LinkOverlay>
+                            </HStack>
+                            <HStack>
+                              <Badge
+                                size="sm"
+                                colorScheme={FILL_COLOR_MAP[fill.state as keyof object] || ''}
+                              >
+                                {FILL_STATE_MAP[fill.state as keyof object] || ''}
```

Comment:
Mentioned elsewhere, but i think order status is more relevant

Reply:
done